packages/cli/config/.eslintrc.js

```diff
@@ -69,6 +69,14 @@ module.exports = {
       ],
 
       rules: {
+        // Extend jsx-a11y/alt-text to also look for Next.js Images
+        "jsx-a11y/alt-text": [2, {
+          "elements": ["img", "object", "area", "input[type=\"image\"]"],
```

Comment:
Documentation wasn't clear if adding new elements could/would erase the current ones, so I added the defaults.